modules/admin/proc-running-rhdh-behind-a-proxy.adoc

```diff
@@ -0,0 +1,115 @@
+[id='proc-running-rhdh-behind-a-proxy_{context}']
+= Running the {product-very-short} application behind a corporate proxy
+
+You can run the {product-very-short} application behind a corporate proxy by setting any of the following environment variables before starting the application:
+
+* `HTTP_PROXY`: Denotes the proxy to use for HTTP requests.
+* `HTTPS_PROXY`: Denotes the proxy to use for HTTPS requests.
+
+Additionally, you can set the `NO_PROXY` environment variable to exclude certain domains from proxying. The variable value is a comma-separated list of hostnames that do not require a proxy to get reached, even if one is specified.
+
+For Helm-based deployment, both a developer or a cluster administrator, with permissions to create resources in the cluster, can configure the proxy variables in a `values.yaml` Helm configuration file.
+
+For Operator-based deployment, the approach you use for proxy configuration is based on your role:
+
+* As a cluster administrator with access to the Operator namespace, you can configure the proxy variables in the Operator's default ConfigMap file. This configuration applies the proxy settings to all the users within the namespace where the Operator is deployed.
```

Comment:
> This configuration applies the proxy settings to all the users within the namespace where the Operator is deployed.

Just a small clarification on this as I think the "within the namespace ..." part can be misleading: I would say something like this: "This configuration applies the proxy settings to all the users of the operator.".

As the operator watches custom resources cluster-wide, the proxy settings configuration (once added by an admin to the operator default config) will be applied to all the users, regardless of the namespace in which they deploy their operator-based RHDH application (not necessarily within the namespace where the operator is deployed).

Reply:
Thanks, @rm3l, for the clarification and making me understand this part :)